strings/tips.txt

```diff
@@ -116,11 +112,11 @@ As an Atmospheric Technician, your ATMOS holofan projector blocks gases while al
 As an Atmospheric Technician, burning a plasma/oxygen mix inside the incinerator will not only produce power, but also gases such as tritium, water vapor and carbon dioxide.
 As an Atmospheric Technician, you can change the layer of a pipe by clicking with it on a wrenched pipe or other atmos component of the desired layer.
 As the Head of Security, you are expected to coordinate your security force to handle any threat that comes to the station. Sometimes it means making use of the armory to handle a blob, sometimes it means being ruthless during a revolution or cult.
-As the Head of Security, you can call for executions or forced cyborgization, but may require the Captain's approval.
+As the Head of Security, you can call for executions or forced cyborgization, but require the Captain's approval.
```

Comment:
typo

Reply:
i thought it was a typo but i couldn't be bothered to read the whole sentence

Reply:
It is a typo. It should be "but requires".